zephyr/CMakeLists.txt

```diff
@@ -253,6 +253,7 @@ zephyr_library_sources(
 	edf_schedule.c
 	ll_schedule.c
 	schedule.c
+	init.c
```

Comment:
Probably best if we use src/platform/name/platform.c here ?

Reply:
@lgirdwood So we can include `src/platform/intel/cavs/platform.c` and also init.c for initialization?

Reply:
yes, we should sue the platform.c (for each platform here), that we we dont need to pull it into Zephyr.

Reply:
Added commit 7f05b90a3ed269798b2b737c36a8bcc3beb9b065